tests/unit_tests/kotahi_docmaps/v1/test_data.py

```diff
@@ -88,14 +88,17 @@
     'manuscript_versions': [MANUSCRIPT_VERSION_1, MANUSCRIPT_VERSION_2]
 }
 
-REVIEW_1 = '''
-    Reviewer #1 (Public Review):
+SPACE_1 = ' '
```

Comment:
I think `SPACE` probably doesn't need a numbered suffix. Or if we did use variations of whitespaces, then we would likely name them.

Reply:
I can remove it. But I thought we are doing it even we don’t have many with same name. When we look at the naming in our tests they always has number suffix.